grails-app/i18n/messages.properties

```diff
@@ -1044,6 +1044,7 @@ kuorum.web.commands.editor.EditorAccountCommand.alias.max.size = Alias max lengt
 kuorum.web.commands.editor.EditorAccountCommand.alias.nullable = Alias not empty if the user is active
 kuorum.web.commands.editor.EditorAccountCommand.alias.unique = Not available alias
 kuorum.web.commands.editor.EditorAccountCommand.bio.label = Description
+kuorum.web.commands.editor.EditorAccountCommand.bio.maxCharsHtml.error.kuorum.web.commands.editor.EditorAccountCommand.bio = Maximum {3} characters with spaces
```

Comment:
This code should be:
kuorum.web.commands.editor.EditorAccountCommand.bio.maxCharsHtml